k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageViewFragment.java

```diff
@@ -269,6 +269,14 @@ public boolean allHeadersVisible() {
         return mMessageView.getMessageHeaderView().additionalHeadersVisible();
     }
 
+    public boolean isAccountReportSpamEnabled() {
+        try {
+            return (mAccount != null && !mAccount.getReportSpamRecipient().isEmpty());
+        } catch (Exception e) {
+            return false;
```

Comment:
I had copied the following functionality from the same file:
`    public boolean isAccountExpungeCapable() {
        try {
            return (mAccount != null && mAccount.getRemoteStore().isExpungeCapable());
        } catch (Exception e) {
            return false;
        }
    }`